accessibility_scanning/README.md

```diff
@@ -0,0 +1,131 @@
+## Accessibility Scanning Using Pa11y-ci
+
+### Introduction
+
+Building a website or application that is easy accessible toeveryone is  not only an important of the user experience, but also a requirement of all federally funded projects. GSA provides a helpful [Section 508](https://section508.gov)  portal. [Pa11y](https://github.com/pa11y/pa11y) is an automation tool that helps you scan your static web pages to check for accessibility problems and errors. It can be setup on your machine locally or remotely using a CI tool.
+
+### Installation and setup(For MacOS)
+
+1. [Concourse CI](https://concourse.ci/index.html) allows you to run multiple compliance scanning jobs  on your machine using a virtual machine. It is highly recommended to go through its [excellent documentation and "Hello World" tutorial ](https://concourse.ci/hello-world.html) before writing custom jobs for your project.
+
+2. If you want to run locally Pa11y-ci you will need to :
+
+   * install **vagrant** and **virtualbox** ](http://sourabhbajaj.com/mac-setup/Vagrant/README.html)
+   * use [The Fly CLI](https://concourse.ci/fly-cli.html) , a command line tool for Concourse which is  used for a number of tasks from connecting to a shell in one of your build's containers to uploading new pipeline configuration into a running Concourse.
+
+3. If you're on MacOS , you will have to `chmod +x` the downloaded binary and put it in your `$PATH`. This can be done in one fell swoop with `install path/to/fly /usr/local/bin/fly`
+
+4. Run Vagrant
+
+   ```sh
+   vagrant init concourse/lite # creates ./Vagrantfile
+   vagrant up                  # downloads the box and spins up the VM
+   ```
+
+5. Login into fly
+
+   `fly -t lite login -c http://192.168.100.4:8080`
+
+   ​
+
+### Running Pa11y-ci on your local machine
+
+1. Create a `.yml` file containing the job with tasks that you want to run
+
+`touch accessibility-scan.yml`
+
+`vim accessibility-scan.yml`
+
+```yml
+jobs:
+  - name: accessibility-scan
+    plan:
+    - task: run-pa11y
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: node
+        run:
+          path: sh
+          args:
+          - -exc
+          - |
+            npm install -g pa11y-ci
+            npm install -g phantomjs
+            mkdir accessibility-scan
+            pa11y-ci --json --sitemap https://18f.gsa.gov/sitemap.xml > accessibility-scan/summary.txt
```

Comment:
To make this a bit clearer, maybe use a domain like "https://your.website.example.gov/sitemap.xml"